pyproject.toml

```diff
@@ -23,7 +23,7 @@ classifiers = [
 keywords = ["aiida", "workflows", "lammps"]
 requires-python = ">=3.8"
 dependencies = [
-    "aiida-core[atomic-tools]>=1.4.0,<2.0.0",
+    "aiida-core[atomic-tools]>=2.0.0,<3.0.0",
```

Comment:
I think the psycop2g-binary pinning can be removed here now?

Reply:
You want me to remove the `psycopg2-binary<2.9` by `psycopg2-binary`? It is just that the diff in the `pyproject.toml` is showing a change in the version of aiida-core that already exists.

Reply:
yep, this was indeed fixed upstream: https://github.com/aiidateam/aiida-core/pull/5104

Reply:
I guess we don't really to explicitly set `psycog-binary2 ` as a dependency right? as it is an `aiida-core` dependency. The same for `ase` as it is installed via `aiida-core[atomic-tools]`.